From 035941f6d042b1d546e707337d776ba869d9d753 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:23:23 -0400
Subject: [cpufreq] Mark function as static in cpufreq.c

Message-id: <1409235879-1283-110-git-send-email-prarit@redhat.com>
Patchwork-id: 89415
O-Subject: [RHEL7.1 PATCH BZ 1134639 109/185] cpufreq: Mark function as static in cpufreq.c
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 8a5c74a175ac117e55a50b489ee041f40ff8866b
Author: Rashika Kheria <rashika.kheria@gmail.com>
Date:   Wed Feb 26 22:12:42 2014 +0530

    cpufreq: Mark function as static in cpufreq.c

    Mark function as static in cpufreq.c because it is not
    used outside this file.

    This eliminates the following warning in cpufreq.c:
    drivers/cpufreq/cpufreq.c:355:9: warning: no previous prototype for 'show_boost' [-Wmissing-prototypes]

    Signed-off-by: Rashika Kheria <rashika.kheria@gmail.com>
    Reviewed-by: Josh Triplett <josh@joshtriplett.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq.c b/drivers/cpufreq/cpufreq.c
index 10a8300..140b29b 100644
--- a/drivers/cpufreq/cpufreq.c
+++ b/drivers/cpufreq/cpufreq.c
@@ -352,7 +352,7 @@ EXPORT_SYMBOL_GPL(cpufreq_notify_post_transition);
 /*********************************************************************
  *                          SYSFS INTERFACE                          *
  *********************************************************************/
-ssize_t show_boost(struct kobject *kobj,
+static ssize_t show_boost(struct kobject *kobj,
      struct attribute *attr, char *buf)
 {
  return sprintf(buf, "%d\n", cpufreq_driver->boost_enabled);
-- 
1.7.1